integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/indexnull/IndexNullAsValueIT.java

```diff
@@ -123,7 +123,8 @@ void indexNullAsValue_fullText() {
 	}
 
 	private void setUp() {
-		setupHelper.start().withIndexes( index ).setup();
+		setupHelper.start( tckBackendHelper -> tckBackendHelper.createHashBasedShardingBackendSetupStrategy( 4 ) )
+				.withIndexes( index ).setup();
```

Comment:
Wouldn't it be simpler to just enforce a single shard in the default strategy?

I can understand being explicit about this in KNN tests, but here... It seems a bit out of place.

Reply:
> Wouldn't it be simpler to just enforce a single shard in the default strategy?

I was thinking about it at first... but then I thought that it may be better to keep the number of shards up to the backend to decide, so this way, we'd test with `1` on "local" Elasticsearch and OpenSearch on CI and with `5` or whatever the default is on AWS... just so that cover more combinations.

Reply:
Ok, let's do it your way :)